### PR TITLE
chore: fixing UI nftsales

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ import nftrole from "./config/nftRole"
 import globalxp from "./config/globalxp"
 import eventxp from "./config/eventxp"
 import log from "./config/log"
+import testdemo from "./renderSalesMessage"
```
Comment (trkhoi) [inline src/commands/index.ts:42]: Remove test cmd

File: src/commands/index.ts [hunk 2]
```diff
@@ -68,6 +69,7 @@ export const originalCommands: Record<string, Command> = {
   top,
   sales,
   // config
+  testdemo,
```
Comment (trkhoi) [inline src/commands/index.ts:72]: Remove test cmd

File: src/commands/renderSalesMessage.ts [hunk 3]
```diff
@@ -1,20 +1,20 @@
 import { Message, MessageAttachment } from "discord.js"
-import { sales } from "types/sales"
+import { NftSales } from "types/sales"
```
Comment (trkhoi) [inline src/commands/renderSalesMessage.ts:2]: Remove test cmd

File: src/types/sales.ts [hunk 4]
```diff
@@ -1,6 +1,8 @@
 export type NftSales = {
   collection: string
+  avatar: string
   name: string
+  userID: string
```
Comment (trkhoi) [inline src/types/sales.ts:5]: What is userId, nft does not hve user information

**Review by trkhoi — CHANGES_REQUESTED** [review]
